crates/top/re_sdk/src/recording_stream.rs

```diff
@@ -1088,6 +1130,19 @@ impl RecordingStream {
         self.log_serialized_batches_impl(row_id, ent_path, static_, comp_batches)
     }
 
+    /// Set the name of the recording.
+    ///
+    /// When provided, this name will show up in the viewer.
+    #[inline]
+    pub fn set_recording_name(&self, name: impl Into<String>) -> RecordingStreamResult<()> {
+        let component = re_types_core::components::RecordingName::from(name.into());
+        self.log_serialized_batches(
```

Comment:
Change to use the archetype API here to avoid untagged component. (`update_fields`)